helper/pdfGeneration.js

```diff
@@ -1076,4 +1076,326 @@ const uploadPdfToCloud = async function(fileName, folderPath) {
      }
  
     })
- }
\ No newline at end of file
+ }
+
+
+ // ============> PDF generation function for withRubics  ======================>
+/**
+ * Generate  PDF observationwith Rubrics.
+ * @method
+ * @name assessmentPdfGeneration
+ * @param {Array} assessmentRes          - Conatins data of chartObject.
+ * @param {String} submissionId          - Id of the observation submission
+ * @returns {Object}                     -  contain PDF url
+ */
+exports.assessmentPdfGeneration = async function assessmentPdfGeneration(
+    assessmentRes,
+    submissionId=""
+  ) {
+    return new Promise(async function (resolve, reject) {
+      let currentTempFolder =
+        "tmp/" +
+        uuidv4() +
+        "--" +
+        Math.floor(Math.random() * (10000 - 10 + 1) + 10);
+  
+      let imgPath = __dirname + "/../" + currentTempFolder;
```

Comment:
@praveenKDass this type of path assign can cause error in server
better to do 
let imgPath = path.resolve(__dirname, '../../', currentTempFolder)
and test. make the change every where. even in @borkarsaish65 files also you check it